README.md

```diff
@@ -162,9 +154,49 @@ To provide Keystone credentials and configure the agent using auto-detection run
 
 For more parameters, please see [Monasca Agent Documentation](https://github.com/openstack/monasca-agent/blob/master/docs/Agent.md#explanation-of-primary-monasca-setup-command-line-parameters).
 
-This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above.
+This will create and run a new service file `/etc/systemd/system/monasca-agent.service`
+with the configuration set as per the arguments mentioned above.
+
+### Uninstalling Metrics agent
+Stop the services:
+```
+systemctl stop monasca-agent
+systemctl disable monasca-agent
+```
 
-### Log agent
+Delete created files:
+```
+sudo rm -rf /etc/monasca/agent/
+sudo rm -rf /opt/monasca-agent/
+sudo rm -rf /etc/sudoers.d/mon-agent/
+```
+Delete the service related to the agent.
+```
+sudo rm -f /etc/systemd/system/monasca-agent.service
```

Comment:
Please change option from `-f` to `-rf`.

Reply:
`-r` for `rm` is for removing recursively directories. Here I only remove one file.